### PR TITLE
[NET] Updated scheme and NUI information

### PR DESCRIPTION
**Pull request by mihashco**
### Change Description ###

* Exposed information about NUI in the top dotnet docs page. 
* Modified scheme to point where NUI is placed in the Tizen frameworks

### CODE REVIEW

File: docs/application/dotnet/learn-about-dotnet.md
Comment (omar-srbd) [inline docs/application/dotnet/learn-about-dotnet.md:33]: For second heading, please use sentence style capitalization
Ref: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
```suggestion
## Tizen .NET features and components
```

File: docs/application/dotnet/learn-about-dotnet.md
Comment (omar-srbd) [inline docs/application/dotnet/learn-about-dotnet.md:62]: Line 62: The hyperlink "TizenFX API" is not working. Can you please check?

**Review by omar-srbd — COMMENTED** [review]
Hello @mihashco,
I have gone through the file in this PR and made suggestions wherever possible.
There were a few lines where I could not comment. I am listing the issues along with their correction suggestions below:

1.	Line 3 - “Tizen .NET is an exciting new way to develop applications for the Tizen operating system, running on 50 million Samsung devices, including TVs, wearables, mobile phones, and many other IoT devices around the world.”
Suggestion – “Tizen .NET is an exciting new way to develop applications for Tizen operating system, running on 50 million Samsung devices, including TVs, wearables, mobile phones, and many other IoT devices around the world.”
Reference – For nouns such as tool names, it is recommended to not use "The"
2.	Line 5 - “The existing Tizen frameworks are either C-based with no advantages of a managed runtime, or HTML5-based with fewer features and lower performance than the C-based solution.”
Suggestion – “The existing Tizen frameworks are either C-based with no advantages of a managed runtime or HTML5-based with fewer features and lower performance than the C-based solution.”
Reference – The comma before ‘or’ seems unnecessary here since the content before and after ‘or’ are dependent clauses.
3.	Line 9 - “Managed Runtime Advantages”
Suggestion – “Managed runtime advantages”
Reference – For second heading, please use sentence style capitalization
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
4.	Line 27 - “Managed runtime helps you to focus on the business logic and the code specific to the application, while reducing the number of errors in the code.”
Suggestion – “Managed runtime helps you to focus on the business logic and the code specific to the application while reducing the number of errors in the code.”
Reference – The comma before ‘while’ doesn’t seem necessary here. Can you please check?
 6.	Line 54 - “The Visual Studio Tools for Tizen enables Tizen support for Xamarin.Forms.”
Suggestion – “Visual Studio Tools for Tizen enable Tizen support for Xamarin.Forms.”
Reference – Since ‘Tools’ is plural here, the correct verb form should be ‘enable’.
For nouns such as tool names, it is recommended to not use "The". Example: Visual Studio Tools
7.	Line 58 - “Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike the Xamarin.Forms which support cross platforms.”
Suggestion – “Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike Xamarin.Forms that support cross platforms.”
Reference – The usage of ‘the’ after ‘unlike’ doesn’t seem necessary here. Can you please check?
9.	Line 70 - “You can create a Tizen .NET application project with the Project Wizard tool in Visual Studio Tools for Tizen.”
Suggestion – “You can create a Tizen .NET application project with Project Wizard tool in Visual Studio Tools for Tizen.”
Reference – For nouns such as tool names, it is recommended to not use "The"
10.	Line 72 - “The Visual Studio Tools for Tizen extension is registered in Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from Visual Studio Marketplace in the Visual Studio IDE.”
Suggestion – “Visual Studio Tools for Tizen extension is registered in Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from Visual Studio Marketplace in the Visual Studio IDE.”
Reference – For nouns such as tool names, it is recommended to not use "The"

File: docs/application/dotnet/learn-about-dotnet.md
Comment (mihashco) [reply docs/application/dotnet/learn-about-dotnet.md:33]: Hello @omar-srbd . This line was not changed by this commit. I can apply your suggestion for overall improvement but it will be great to fix only changed lines...

**Comment by mihashco** [comment]
> Hello @mihashco, I have gone through the file in this PR and made suggestions wherever possible. There were a few lines where I could not comment. I am listing the issues along with their correction suggestions below:
> 
> 1. Line 3 - “Tizen .NET is an exciting new way to develop applications for the Tizen operating system, running on 50 million Samsung devices, including TVs, wearables, mobile phones, and many other IoT devices around the world.”
>    Suggestion – “Tizen .NET is an exciting new way to develop applications for Tizen operating system, running on 50 million Samsung devices, including TVs, wearables, mobile phones, and many other IoT devices around the world.”
>    Reference – For nouns such as tool names, it is recommended to not use "The"
> 2. Line 5 - “The existing Tizen frameworks are either C-based with no advantages of a managed runtime, or HTML5-based with fewer features and lower performance than the C-based solution.”
>    Suggestion – “The existing Tizen frameworks are either C-based with no advantages of a managed runtime or HTML5-based with fewer features and lower performance than the C-based solution.”
>    Reference – The comma before ‘or’ seems unnecessary here since the content before and after ‘or’ are dependent clauses.
> 3. Line 9 - “Managed Runtime Advantages”
>    Suggestion – “Managed runtime advantages”
>    Reference – For second heading, please use sentence style capitalization
>    https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#capitalization
> 4. Line 27 - “Managed runtime helps you to focus on the business logic and the code specific to the application, while reducing the number of errors in the code.”
>    Suggestion – “Managed runtime helps you to focus on the business logic and the code specific to the application while reducing the number of errors in the code.”
>    Reference – The comma before ‘while’ doesn’t seem necessary here. Can you please check?
> 5. Line 54 - “The Visual Studio Tools for Tizen enables Tizen support for Xamarin.Forms.”
>    Suggestion – “Visual Studio Tools for Tizen enable Tizen support for Xamarin.Forms.”
>    Reference – Since ‘Tools’ is plural here, the correct verb form should be ‘enable’.
>    For nouns such as tool names, it is recommended to not use "The". Example: Visual Studio Tools
> 6. Line 58 - “Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike the Xamarin.Forms which support cross platforms.”
>    Suggestion – “Tizen.Wearable.CircularUI APIs are supported only on Tizen wearable devices with Tizen .NET support, unlike Xamarin.Forms that support cross platforms.”
>    Reference – The usage of ‘the’ after ‘unlike’ doesn’t seem necessary here. Can you please check?
> 7. Line 70 - “You can create a Tizen .NET application project with the Project Wizard tool in Visual Studio Tools for Tizen.”
>    Suggestion – “You can create a Tizen .NET application project with Project Wizard tool in Visual Studio Tools for Tizen.”
>    Reference – For nouns such as tool names, it is recommended to not use "The"
> 8. Line 72 - “The Visual Studio Tools for Tizen extension is registered in Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from Visual Studio Marketplace in the Visual Studio IDE.”
>    Suggestion – “Visual Studio Tools for Tizen extension is registered in Visual Studio Marketplace. You can [install the extension](../vstools/install.md) from Visual Studio Marketplace in the Visual Studio IDE.”
>    Reference – For nouns such as tool names, it is recommended to not use "The"

Hello @omar-srbd . Thank you for review. Could you please do review only on changed lines? I am not the author of the lines listed and it would be better if I only corrected the changes made by me.

**Review by omar-srbd — APPROVED** [review]
I have reviewed both the files in the PR and provided correction suggestions. However, the author requested to review only the parts that he has modified.
After another check on the modification provided by the author, I found no issues. I discussed this matter with @mijin85cho as well.
I am approving this PR for merging after getting the go-ahead from her.

**Pull request merged**
